pyproject.toml

```diff
@@ -1,5 +1,5 @@
 [build-system]
-requires = ["setuptools", "wheel"]
+requires = ["setuptools", "wheel", "setuptools_scm"]
```

Comment:
The example at https://pypi.org/project/setuptools-scm/ says "Note that the ``toml`` extra must be supplied.":
```
requires = ["setuptools>=42", "wheel", "setuptools_scm[toml]>=3.4"]
```
Maybe it works fine without it in most cases but maybe not all?

Reply:
That's if you are using pyproject.toml to configure the file.